src/monodroid/jni/config.h

```diff
@@ -4,8 +4,6 @@
 #include <monodroid.h>
 #include <stdint.h>
 
-MONO_API  int CreateNLSocket (void);
-MONO_API  int ReadEvents (void *sock, void *buffer, int32_t count, int32_t size);
-MONO_API  int CloseNLSocket (void *sock);
+// This file currently does nothing, but is needed for builds to succeed.
```

Comment:
Incorrect comment -- it isn't doing nothing, it has `#include`.

Reply:
Fair point.